metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/UserMetadataService.java

```diff
@@ -80,6 +81,13 @@ default void softDeleteDataMetadatas(String userId, List<String> uris) {
     default void deleteDefinitionMetadatas(List<QualifiedName> names) {
```

Comment:
Why are we adding business metadata API's to a `UserMetadataService`?

Reply:
ya, that's the thing. UserMetadataService is 'metadata' service.